client/src/assets/scss/theme.scss

```diff
@@ -46,25 +46,14 @@ $BCgovAGold2: #FFF8EF;
 $BCgovAGold3: #FFE0BC;
 $BCgovAGold4: #FFD4A2;
 
-$border: #899EB2;
+// Font colors
+$dk-text: $gray9;
+$text: $gray7;
 
-// Link
-$link: #1669bb;
+// Some popular colors
+$app-blue: #1669bb;
+$app-red: #d3272c;
+$app-green: #2e8540;
 
-// Approved
 $approved: #1a9031;
```

Comment:
Hmm, we have 2 greens for the same thing.
Tracey spec'ed out "app-green" using that hex code. Did she also spec out "approved" colour? We should ask her for a single approved green colour.

Reply:
Fixed.